<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>statisticsPage</class>
 <widget class="QWidget" name="statisticsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>449</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>借阅情况</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <column>
          <property name="text">
           <string>图书</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>借书人</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>借书日期</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>还书日期</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelBorrowed">
         <property name="text">
          <string>总借阅量：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelInLibrary">
         <property name="text">
          <string>在馆图书量：</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
